### DISCUSSION TITLE
Simplify Sign Out process

### DISCUSSION THREAD

**Discussion by cameron0208** [community/community#6453] [Mobile] [answered]
The current process of Profile > Manage Accounts > Edit > Tap account > Tap icon > Tap 'Sign Out' in account list > Tap 'Sign Out' in prompt @ bottom of screen is extremely tedious and is not intuitive.

**Comment by amrehman** [comment, marked as answer]
Thank you so much for the request and sorry to hear that you find the current sign out flow tedious. We will discuss this amongst our team and come up with a better experience for you! Thanks!

**Reply by gorkazl** [reply]
I very much agree. I only want the mobile app for the 2-factor authentication on desktop, and sign out inmediately. If I am always logged into the account and my phone is stolen, they gain complete access to my account and projects. I don't see how this is secure.

**Comment by willi84** [comment]
Would be awesome if I can easily switch between github accounts (e.g. work and private), especially to add this app as 2FA option too.
